java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -5890,4 +5892,104 @@ public void test_dump_restore_withOptions(BaseClient client) {
                                         .get());
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void sscan(BaseClient client) {
+        String key1 = "{key}-1" + UUID.randomUUID();
+        String key2 = "{key}-2" + UUID.randomUUID();
+        long initialCursor = 0;
+        long defaultCount = 10;
+        String[] numberMembers = new String[125];
+        for (int i = 0; i < numberMembers.length; i++) {
+            numberMembers[i] = String.valueOf(i);
+        }
+        String[] charMembers = new String[] {"a", "b", "c", "d", "e"};
+        int resultCursorIndex = 0;
+        int resultCollectionIndex = 1;
+
+        // Empty set
+        Object[] result = client.sscan(key1, initialCursor).get();
+        assertEquals(String.valueOf(initialCursor), result[resultCursorIndex]);
+        assertDeepEquals(new String[] {}, result[resultCollectionIndex]);
+
+        // Negative cursor
+        result = client.sscan(key1, -1).get();
+        assertEquals(String.valueOf(initialCursor), result[resultCursorIndex]);
+        assertDeepEquals(new String[] {}, result[resultCollectionIndex]);
+
+        // Result contains the whole set
+        assertEquals(charMembers.length, client.sadd(key1, charMembers).get());
+        result = client.sscan(key1, initialCursor).get();
+        assertEquals(String.valueOf(initialCursor), result[resultCursorIndex]);
+        assertEquals(charMembers.length, ((Object[]) result[resultCollectionIndex]).length);
```

Comment:
Unable to check that the arrays match exactly as the result is not necessarily always in the same order.

Reply:
Convert arrays to sets to compare